quickstarts/ansible-automation-controller/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: ansible-automation-controller
+description: |
+  ## Why monitor Ansible Automation Controller?
```

Comment:
```suggestion
  ## Why monitor your Ansible Automation Controller?
```

Reply:
Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.